src/components/ui/Panel.svelte

```diff
@@ -7,12 +7,12 @@
   export let padHeader: boolean = true;
 </script>
 
-<div class="panel" class:borderLeft class:borderRight class:borderTop>
-  <div class="header" class:padHeader>
+<div class="panel" class:border-left={borderLeft} class:border-right={borderRight} class:border-top={borderTop}>
```

Comment:
Not as Svelty, but I can live with it

Reply:
i know...it did hurt a little making it this way...